packages/vue-instantsearch/src/util/vue-compat/index-vue2.js

```diff
@@ -9,22 +9,22 @@ export { Vue, Vue2, isVue2, isVue3, version };
 
 const augmentCreateElement =
   (createElement) =>
-  (tag, propsWithClassName = {}, children) => {
+  (tag, propsWithClassName = {}, ...children) => {
```

Comment:
I think we need a test or explanation for this change

Reply:
Yeah I meant to do that, basically we didn't support the case where we'd call `createElement` with children as spread arguments.
With the introduction of the banner, this code : 
```jsx
<div>
  {a && <A />}
  {b && <B />}
</div>
```
Gets transformed to : 
```js
createElement('div', {}, a && createElement(A), b && createElement(B))
```
Which isn't something we ever supported, so this is more of a bug fix I guess.

Reply:
sounds good, maybe it should even be a separate PR though?